test/test_helper.rb

```diff
@@ -55,14 +63,12 @@ def self.capture_all
     let(:stdout) { StringIO.new }
     let(:stderr) { StringIO.new }
 
-    around do |t|
+    before do |_t|
```

Comment:
FYI don't need the `_t`
```suggestion
    before do
```